Pipfile

```diff
@@ -20,9 +20,12 @@ sphinx = "*"
 sphinxcontrib-django = "*"
 sphinx-last-updated-by-git = "*"
 sphinx-rtd-theme = "*"
+reportlab = "*"
+xhtml2pdf = "*"
 
 [packages]
 integreat-cms = {editable = true, path = "."}
+zipp = {editable = true, path = "."}
```

Comment:
```suggestion
```
This should not be there, I think this is a bug of Pipenv which adds this package as local package when new pip packages are added.
You can circumvent this problem if you delete the directories `build`, `.venv` and `src/integreat_cms.egg-info/` before doing the `pipenv install`.